### PR TITLE
Add golangci-lint step to Github workflows

### PR DESCRIPTION
**Pull request by 0michalsokolowski0**

### CODE REVIEW

File: sqlite.go [hunk 1]
```diff
@@ -48,14 +48,6 @@ func (sw *SQLiteResultItemWriter) createTable() error {
 	return err
 }
 
-func min(a, b int) int {
```
Comment (0michalsokolowski0) [inline sqlite.go:51]: sqlite.go:51:6: `min` is unused (deadcode)
func min(a, b int) int {

**Comment by 0michalsokolowski0** [comment]
I see it finally worked 👍🏻

**Comment by eatonphil** [comment]
Yeah I forgot how flaky it is with low timeout.

**Comment by eatonphil** [comment]
Thank you!

**Pull request merged**